flagsmith-core.ts

```diff
@@ -835,7 +835,8 @@ const Flagsmith = class {
     private updateEventStorage() {
         if (this.enableAnalytics) {
             const events = JSON.stringify(this.getState().evaluationEvent);
-            AsyncStorage!.setItem(FlagsmithEvent, events);
+            AsyncStorage!.setItem(FlagsmithEvent, events)
+            .catch((e) => this.log("Exception setting item in async storage", e));
```

Comment:
Should this log an error instead of a regular message?

Also, should this call the `onError` handler?

Reply:
I'm not sure we should call `onError` because this method is only handling analytics and not the main purpose of `getValue` that is returning a value of a feature or a fallback value

Reply:
That said, it makes sense to be a simple console.error instead of this.log